emantomo/__init__.py

```diff
@@ -24,165 +22,15 @@
 # *  e-mail address 'scipion@cnb.csic.es'
 # *
 # **************************************************************************
-
-import os
-import subprocess
-
-import pwem
-import pyworkflow.utils as pwutils
-from scipion.install.funcs import VOID_TGZ
-
-import emantomo.constants as emanConst
-
-__version__ = "3.1.5"
+__version__ = "3.2.0"
 _logo = "eman2_logo.png"
 _references = ['GALAZMONTOYA2015279', 'BELL201625']
 _url = "https://github.com/scipion-em/scipion-em-emantomo"
 
+try:
+    import eman2
+    class Plugin(eman2.Plugin):
+        pass
```

Comment:
I'd implement here _defeineBinaries as an empty method to avoid double definition of binaries